Resources/Private/Fusion/Documents/Document.Menu.fusion

```diff
@@ -0,0 +1,70 @@
+prototype(Networkteam.Neos.Mockup:Document.Menu) < prototype(Neos.Fusion:Value) {
```

Comment:
If you use a `Neos.Fusion:Component` you dont have to push the api-values to the context manually